v1/linkml-schemas/address.yaml

```diff
@@ -18,7 +17,12 @@ default_range: string
 classes:
   PostalAddressClass:
     description: >-
-      A postal address associated with a user, returned by an Address Lookup API or by manual data input.  
+      A postal address associated with a user, returned by an Address Lookup API or by manual data input.
+      <br />
+      <br />
+      JSON schema - [di_vocab:PostalAddress](json-schemas/PostalAddress.json) 
+      <br />
+      Examples - [Example Postal Address Files](/#postal-address-examples) 
```

Comment:
I wonder if it'd be more scalable to have the links to each example here, and have the home page point at this one. Otherwise the home page will end up getting bigger and bigger.